conn.go

```diff
@@ -1243,28 +1243,25 @@ type MultiResponse struct {
 	Error  error
 }
 
+var (
+	_ operation = &CreateRequest{}
```

Comment:
A better (although maybe less clear) way to do this is

```
	_ operation             = (*CreateContainerRequest)(nil)
```

This way no memory is allocated (although i am not sure if the compiler optimizes this away anyway?

Reply:
So since this a pointer it should be a nil type. 

https://github.com/uber-go/guide/blob/master/style.md#verify-interface-compliance

> The right hand side of the assignment should be the zero value of the asserted type. This is nil for pointer types (like *Handler), slices, and maps, and an empty struct for struct types.